### PR TITLE
fr translation

### PR DESCRIPTION
**Pull request by AugustinMauroy**
# fr translation
all file translated in French

### CODE REVIEW

File: fr/proposals.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "authors": "Auteurs",
+  "champion": "Champion",
+  "champions": "Champions",
+  "authors-and-champions": "Auteurs et champions",
+  "see-more": "Voir les propositions à toutes les étapes",
+  "needs_translation": "Besoin de traduction",
+  "needs_translation_url": "https://github.com/tc39/tc39.github.io/blob/master/translation.md",
+  "community_translations": "Cette page est traduite par la communauté. Si vous souhaitez contribuer, lisez nos <a href=\"https://github.com/tc39/tc39.github.io/blob/master/translation.md\">directives</a>.",
+  "months": {
+    "1": "Janvier",
```
Comment (DerekNonGeneric) [inline fr/proposals.json]:
```suggestion
     "1": "Janvier",
```

Comment (DerekNonGeneric) [reply fr/proposals.json]: You may need to decide whether months are capitalized or not&hellip;

**Comment by AugustinMauroy** [comment]
@codebytere Can you review ?

File: fr/proposals.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "author": "Auteur",
+  "authors": "Auteurs",
+  "champion": "Champion",
+  "champions": "Champions",
+  "authors-and-champions": "Auteurs et champions",
+  "see-more": "Voir les propositions à toutes les étapes",
+  "needs_translation": "Besoin de traduction",
+  "needs_translation_url": "https://github.com/tc39/tc39.github.io/blob/master/translation.md",
+  "community_translations": "Cette page est traduite par la communauté. Si vous souhaitez contribuer, lisez nos <a href=\"https://github.com/tc39/tc39.github.io/blob/master/translation.md\">directives</a>.",
+  "months": {
```
Comment (DerekNonGeneric) [inline fr/proposals.json:30]: Still interested in this one open question please: maybe somebody knows and can perhaps simply get back to me on one thing.

> Should months all begin with a capital letter?

I have never learned any of this in an academic setting, but am probably the only person who maybe can review, so really sorry that this got closed.

Comment (DerekNonGeneric) [inline fr/proposals.json:30]: Still interested in this one open question please: maybe somebody knows and can perhaps simply get back to me on one thing.

> Should months all begin with a capital letter?

I have never learned any of this in an academic setting, but am probably the only person (maybe) who can review, so really sorry that this got closed.

File: translation_fr.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Translation Guidelines
+
+The translation of the TC39 web page aims to make the work of TC39 more accessible to people whose
```
Comment (ctcpip) [inline translation_fr.md:3]: is this file meant to be written in English?

Comment (AugustinMauroy) [reply translation_fr.md:3]: I think English would be better.

**Comment by ljharb** [comment]
@ctcpip Since the PR's head repo has been deleted, this PR can never be updated, so I think it would need to be opened fresh and this PR left closed.

File: fr/intro.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "TC39": {
+    "title": "TC39",
+    "desc": "Le TC39 d'Ecma International est un groupe de développeurs JavaScript, d'implémenteurs, d'universitaires et autres, qui collaborent avec la communauté pour maintenir et faire évoluer la définition de JavaScript.",
+    "ecma-logo": "Nous faisons partie de <br/> <a class=\"ecma-logo-box\" href=\"https://www.ecma-international.org/\"> <img src=\"/assets/img/ecma-logo.svg\" alt=\"Ecma International\" /> </a>"
+  },
+  "contribute": {
+    "title": "Contribuez à",
+    "desc": "Le TC39 accueille les contributions de la communauté JavaScript, qu'il s'agisse de commentaires sur des propositions existantes, d'amélioration de la documentation, de tests, d'implémentations ou même d'idées de fonctionnalités du langage. Voir notre <a href=\"https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md\">contributor guide</a> pour les détails.",
```
Comment (ThomasDq) [inline fr/intro.json:9]: <a href=\"https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md\">guide pour contributeurs</a>

File: fr/intro.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "join-ecma": "Participer aux réunions du TC39 en tant que membre, <a href=\"https://ecma-international.org/memento/join.htm\">rejoindre l'Ecma</a>."
+  },
+  "specs": {
+    "title": "Specs",
+    "desc": "Nous développons la spécification JavaScript (formellement, ECMAScript) <a href=\"https://github.com/tc39/ecma262/\">sur GitHub</a> et se réunissent tous les deux mois pour discuter des propositions. Pour en savoir plus sur le processus, veuillez consulter le <a href=\"https://tc39.es/process-document/\">cinq étapes</a> for new <a href=\"https://github.com/tc39/proposals/\">propositions de fonctionnalités linguistiques</a>. Voir notre <a href=\"https://github.com/tc39/agendas/\">agendas des réunions</a> et <a href=\"https://github.com/tc39/notes/\">minutes</a> pour apprendre plus."
```
Comment (ThomasDq) [inline fr/intro.json:14]: Nous développons la spécification JavaScript (formellement, ECMAScript) <a href=\"https://github.com/tc39/ecma262/\">sur GitHub</a> et nous réunissons tous les deux mois pour discuter des propositions.

File: fr/proposals.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Candidats actuels pour la spécification",
+  "description": "Cette section présente les propositions qui sont à l'étape 3 de <a href=\"https://tc39.es/process-document/\">nos processus</a>, ce qui signifie qu'ils sont presque terminés.",
```
Comment (ThomasDq) [inline fr/proposals.json:3]: ce qui signifie qu'elles sont presque terminées

File: fr/proposals.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+     "3": "mars",
+     "4": "Avril",
+     "5": "Mai",
+     "6": "juin",
+     "7": "juillet",
+     "8": "août",
+     "9": "septembre",
+     "10": "Octobre",
+     "11": "novembre",
+     "12": "Décembre"
```
Comment (ThomasDq) [inline fr/proposals.json:42]: Month camel casing isn't consistent

File: fr/site.json [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  "navigation": {
+    "TC39": {
+      "title": "TC39",
+      "agenda": "Agenda à venir",
+      "notes": "Notes de réunions récentes",
+      "sub-menu": "Basculer le sous-menu pour TC39"
+    },
+    "specs": {
+      "title": "Specs",
+      "proposals": "Active Proposals",
```
Comment (ThomasDq) [inline fr/site.json:13]: Propositions actives

File: fr/site.json [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+      "read": "Lecture de la spécification",
+      "discourse": "Rejoignez le débat",
+      "sub-menu": "Basculer le sous-menu pour Contribuer"
+    },
+    "state-of-proposals": "État des propositions",
+    "skip-content": "Aller au contenu",
+    "open": "Ouvrir le menu"
+  },
+  "footer": {
+    "connect": "Connect",
```
Comment (ThomasDq) [inline fr/site.json:33]: "Connecter"

File: fr/site.json [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    "standards": "Standards",
+    "proposals": "Proposals",
```
Comment (ThomasDq) [inline fr/site.json:35]: "Propositions"

File: fr/site.json [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    "archives": "Archives",
+    "process": "Process",
```
Comment (ThomasDq) [inline fr/site.json:37]: Procedure

File: fr/site.json [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    "coc": "Code of Conduct",
```
Comment (ThomasDq) [inline fr/site.json:38]: Code de Conduite

File: fr/site.json [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+    "feedback": "Nous sommes à la recherche de commentaires.",
+    "feedback-prompt": "S'il vous plaît <a href=\"https://github.com/tc39/tc39.github.io/issues/new\" target=\"_blank\">ouvrir une issue</a> dans <a href=\"https://github.com/tc39/tc39.github.io\" target=\"_blank\">le dépôt de notre site</a>."
```
Comment (ThomasDq) [inline fr/site.json:40]: -ouvrir
+ouvrez

File: fr/stage3.json [hunk 14]
```diff
@@ -0,0 +1,62 @@
+{
+  "proposal-regexp-legacy-features": {
+    "title": "Fonctionnalités RegExp héritées en JavaScript",
+    "description": "Il s'agit d'un projet de spécification pour les fonctionnalités RegExp héritées (obsolètes) en JavaScript, c'est-à-dire les propriétés statiques du constructeur comme <code>RegExp.$1</code> ainsi que le <code>RegExp.prototype.compile</code> méthode."
```
Comment (ThomasDq) [inline fr/stage3.json:4]: ainsi que la méthode <code>RegExp.prototype.compile</code>.

File: fr/stage3.json [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  "proposal-hashbang": {
+    "title": "Hashbang Grammar",
```
Comment (ThomasDq) [inline fr/stage3.json:7]: Grammaire HashBang

File: fr/stage3.json [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+    "description": "Cette proposition vise à correspondre à l'usage de facto de certains hôtes JS CLI qui autorisent les Shebangs / Hashbang. Ces hôtes enlèvent le hashbang afin de générer des textes sources JS valides avant de les transmettre aux moteurs JS actuels. Cela permettrait d'unifier et de standardiser la façon dont cela est fait."
+  },
+  "proposal-atomics-wait-async": {
+    "title": "<code>Atomics.waitAsync</code>",
+    "description": "Une proposition pour un &ldquo;asynchronous atomic wait&rdquo ; pour ECMAScript, principalement pour une utilisation dans les agents qui ne sont pas autorisés à bloquer."
+  },
+  "proposal-relative-indexing-method": {
+    "title": "<code>.at()</code>",
+    "description": "A proposal to add a <code>.at()</code> method to all the basic indexable classes (<code>Array</code>, <code>String</code>, <code>TypedArray</code>)."
```
Comment (ThomasDq) [inline fr/stage3.json:16]: Une proposition pour ajouter une méthode <code>.at()</code> à toutes les classes indexables de base(<code>Array</code>, <code>String</code>, <code>TypedArray</code>).

**Comment by ctcpip** [comment]
closing in favor of #314 

applied suggested changes there

**Pull request closed without merging**